### PR TITLE
sw ix

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-12-19-sw-ix-review.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[Obviously heavy spoilers for the movie and SW 1 to 9]
+
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md:15]: love - > low

File: _posts/2019-12-19-sw-ix-review.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+Coming in I had really love expectations, unfortunately that was not enough.
+
+The movie at the same level of shittiness as [The Force Awakens][1] and way worse than The Last Jedi. It seems like Disney doesn't care at all about existing lore.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: was at?

File: _posts/2019-12-19-sw-ix-review.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+They made a plot and rammed it through not caring about existing characters arcs or existing lore. I see a lot of similarities to Game Of Thrones Season 8 in there, but it is not as bad as GoT in my opinion.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: characters - > character

Comment (MsSlacker) [reply _posts/2019-12-19-sw-ix-review.md]: Already said lore, no need to repeat?

File: _posts/2019-12-19-sw-ix-review.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+Where to start... so many things wrong with this movie.
+
+## The whole SW story arc is nullified
+
+George Lucas has said repeatedly that the whole arc of SW 1 to 6 was the fall to Anakyn to the dark side and its redemption and even if you don't like the prequels (I love them) you can see how they are consistent with the overarching story and rules and how they deliver on that vision. The whole reason for his fall was trying to save Padme. The new trilogy now make it seem that bringing someone back from death is a joke.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: AnakYn?

Comment (MsSlacker) [reply _posts/2019-12-19-sw-ix-review.md]: It's? He's an it? Or the dark side is getting redeemed lol

Comment (MsSlacker) [reply _posts/2019-12-19-sw-ix-review.md]: Too many "and"

File: _posts/2019-12-19-sw-ix-review.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+The whole idea of the chosen one that would bring balance to the force that went vindicated in episode 6 is now in shambles. Anakyn in the end did not bring balance. He failed to kill the emperor. What a joke.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: Force is written with capital F

Comment (MsSlacker) [reply _posts/2019-12-19-sw-ix-review.md]: Y again? It's A N A K I N

File: _posts/2019-12-19-sw-ix-review.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+## No bigger plot
+
+SW 7 to 9 don't have an overline structure. They just decided 3 plots for the 3 movies and that's it. They have characters in common, are set in the same universe, and they happen chronologically but apart from that there is not much else going on.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: Consistent structure?

File: _posts/2019-12-19-sw-ix-review.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+Many story arcs started in previous movies were never finished and were just left abandoned (Fin story arc for example) and you can't see an overall story progression from 7 to 9 as you do in 1 to 3 and 4 to 6.
+
+In the end we are at the end of 9 in the same place as we were in 6, the story didn't not progress at all. Apart from the fact that most people we cared about are now dead. (RIP Carrie Fisher)
+
+## Force as a superpower
+
+The force has always been very strong in what it could do, but it had its limitations. We always knew what it could do (pull, push, lightnings, ..) and what it could not (teleport). Already in The Last Jedi we've seen the force gaining the ability of making someone appearing on another planet and interact with people and things there. We've seen force ghosts before so this could be excused and I didn't think too much ill of it. In this movie those new abilities just keep piling up and up in an unbeliavable fashion. The authors seem to excuse themselves behind the curtain of "it's because of the bond between Kylo and Rei", but even with the bond this is totally crazy.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: appearing - > appear

Comment (MsSlacker) [reply _posts/2019-12-19-sw-ix-review.md]: Rei? R E Y

Comment (MsSlacker) [reply _posts/2019-12-19-sw-ix-review.md]: These kinds of bonds usually were with master/padawan... something that took time to develop... not just two random people -. -

File: _posts/2019-12-19-sw-ix-review.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+A few egregious examples:
+
+1. Giving life
+
+As I said before the whole Anakyn story arc has been destroyed. Now anyone can heal. Force heal is not a new thing, but before it was something really hard to do. Now even bringing people back to life can be done and the whole Darth Plaiguis the Wise thing has disappeared. In the old republic with Jedi in peace and studying for millenia they were never able to crack it, not Windu, not Yoda, but we are to believe than now magically it can be done so easily.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: P L A G U E I S

File: _posts/2019-12-19-sw-ix-review.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+2. Dementors
+
+Now Palpatine has become a dementor. Stealing life force is acceptable and if this was the only thing happening in this movie I would not have complained, it is when it's put together with all the rest that makes you think wtf.
+
+3. Force lightning
+
+Force lightning can now destroy a whole fleet. That same force lightning cannot kill one single Jedi. And can beoverpowered by 2 lightsabers, neat!
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: be overpowered

File: _posts/2019-12-19-sw-ix-review.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+
+## Palpatine
+
+The whole Palpatine thing was ridiculous. He disappeared for 30 years and now he comes back out of knowhere, no forthshadowing at all. He gets introduces in the scrolling credits... And when he does he lasts less than three hours and is dead by the end of the movie.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: knowhere - > nowhere

Comment (MsSlacker) [reply _posts/2019-12-19-sw-ix-review.md]: introduces - > introduced

File: _posts/2019-12-19-sw-ix-review.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+Palpatine has always been a real strategist, especially during the Original Trilogy as we've seen him rise to power. His behaviour here is totally not in line with the character. After having his strenght restored and neutralizing a whole fleet by himself he gets beaten by 2 lightsabers reflecting his lightning back to him in a flashback of Episode 3... We didn't even get a decent fight scene.
+
+## Copy of Ep 6
+
+This is not as egregious as The Force Awakens in which almost the whole movie was a copy paste from episode 4, but the final scene of the vault opening to reveal the rebel fleet losing and Ray having to take the throne to save her friends was so cringy. Try to make it look like you're at least trying...
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: R E Y

File: _posts/2019-12-19-sw-ix-review.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+## Others
+
+- Lando doesn't do anything and doesn't add anything to the movie, was nice to see him back though. They had him go around the galaxy asking for help with Chewy...[^1]
+
+- Fin needs to say something to Rae and that whole plot is abandoned.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: R E Y

File: _posts/2019-12-19-sw-ix-review.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+
+- The knights Kylo of appear out of knowhere and go nowhere. They don't even have energy sabers, what they have looks like can be cut off by any lightsaber with ease.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: knights Kylo of - the K N I G H T S  O F  R E N

File: _posts/2019-12-19-sw-ix-review.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+
+- In the end Ray goes to Tatooine. I would understand if Kylo went there to live like Obi Wan, but Tatooine doesn't mean anything for Ray... They talked about it like the Skywalker loved it. I'm sure both Anakyn and Luke hated that place so much (for obvious reasons).
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: R E Y

Comment (MsSlacker) [reply _posts/2019-12-19-sw-ix-review.md]: It was to honor them (mostly Luke) for what I understood?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-12-19-sw-ix-review.md (hunk 6 above)
Comment (valeIT) [reply _posts/2019-12-19-sw-ix-review.md]: It is not that it was not consistent it was not there AT ALL. Way worse

File: _posts/2019-12-19-sw-ix-review.md (hunk 14 above)
Comment (valeIT) [reply _posts/2019-12-19-sw-ix-review.md]: > It was to honor them (mostly Luke) for what I understood?

In person reply xD

File: _posts/2019-12-19-sw-ix-review.md [hunk 15]
```diff
@@ -13,72 +13,94 @@ author: Valentino Urbano
...
+George Lucas has said repeatedly that the whole arc of SW 1 to 6 was the fall of Anakin to the dark side and his redemption. Even if you don't like the prequels (I love them) you can see how they are consistent with the overarching story and rules and how they deliver on that vision.
 
-The whole idea of the chosen one that would bring balance to the force that went vindicated in episode 6 is now in shambles. Anakyn in the end did not bring balance. He failed to kill the emperor. What a joke.
+The whole reason for his fall was trying to save Padme. The new trilogy now make it seem that bringing someone back from death is a joke.
+
+The whole idea of the chosen one that would bring balance to the Force that culminated in Vader throwing the Emperor down a shaft in episode 6 is now in shambles. Anakin in the end did not bring balance. He failed to kill the emperor. What a joke. It was all for nothing.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: emperor uppercase, you just wrote it uppercase last sentence

File: _posts/2019-12-19-sw-ix-review.md [hunk 16]
```diff
@@ -13,72 +13,94 @@ author: Valentino Urbano
...
 
 ## No bigger plot
 
-SW 7 to 9 don't have an overline structure. They just decided 3 plots for the 3 movies and that's it. They have characters in common, are set in the same universe, and they happen chronologically but apart from that there is not much else going on.
+SW 7 to 9 don't have an underline structure at all. They just decided 3 plots for the 3 movies and that's it. They have characters in common, are set in the same universe, and they happen chronologically but apart from that there is not much else going on.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: underline - > no idea what's that supposed to mean so "cohesive"

File: _posts/2019-12-19-sw-ix-review.md [hunk 17]
```diff
@@ -13,72 +13,94 @@ author: Valentino Urbano
...
+
+The authors seem to excuse themselves behind the curtain of "it's because of the bond between Kylo and Rey", but even with that bond this is totally crazy.[^1]
 
 A few egregious examples:
 
 1. Giving life
 
-As I said before the whole Anakyn story arc has been destroyed. Now anyone can heal. Force heal is not a new thing, but before it was something really hard to do. Now even bringing people back to life can be done and the whole Darth Plaiguis the Wise thing has disappeared. In the old republic with Jedi in peace and studying for millenia they were never able to crack it, not Windu, not Yoda, but we are to believe than now magically it can be done so easily.
+As I said before the whole Anakin story arc has been destroyed. Now anyone can heal. Force heal is not a new thing, but before it was something really hard to do. Now even bringing people back to life can be done and the whole Darth Plagueis the Wise thing has disappeared. In the old republic with Jedi in peace and studying for millenia they were never able to crack it, not Windu, not Yoda, but we are to believe than now magically it can be done so easily.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: Can uppercase Old Republic too

File: _posts/2019-12-19-sw-ix-review.md [hunk 18]
```diff
@@ -13,72 +13,94 @@ author: Valentino Urbano
...
+This is not as egregious as The Force Awakens in which almost the whole movie was a copy paste from episode 4, but the final scene of the vault opening to reveal the rebel fleet losing and Rey having to take the throne to save her friends was so cringy. Try to make it look like you're at least trying...
 
 ## Others
 
-- Lando doesn't do anything and doesn't add anything to the movie, was nice to see him back though. They had him go around the galaxy asking for help with Chewy...[^1]
+- Lando doesn't do anything and doesn't add anything to the movie, was nice to see him back though. They had him go around the galaxy asking for help with Chewy...[^2]
+
+- Fin needs to say something to Rey and that whole plot is abandoned.
+
+- The Knights Of Ren of appear out of nowhere and go nowhere. They don't even have energy sabers, what they have looks like can be cut off by any lightsaber with ease.
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: what's the "of appear"?

File: _posts/2019-12-19-sw-ix-review.md [hunk 19]
```diff
@@ -13,72 +13,94 @@ author: Valentino Urbano
...
 
-- Fin needs to say something to Rae and that whole plot is abandoned.
+- Leia is now a randomly a Jedi. As with the rest of the movie without any kind of foreshadowing or explanation. She just is.
 
-- The knights Kylo of appear out of knowhere and go nowhere. They don't even have energy sabers, what they have looks like can be cut off by any lightsaber with ease.
+- All the Star Destroyers that are so advanced to kill a planet with one shot can't lift themselves without being guided by a navigation beacon. A beacon that can be destroyed by laser fire. But there is a second beacon on the command ship, but nowhere else because plot.
 
-- In the end Ray goes to Tatooine. I would understand if Kylo went there to live like Obi Wan, but Tatooine doesn't mean anything for Ray... They talked about it like the Skywalker loved it. I'm sure both Anakyn and Luke hated that place so much (for obvious reasons).
+- Many more (Kylo and Rey kissing, non Jedi surviving in the vacuum of space, Palpatine has manifestations of all the Siths watching the battle while the Jedi are nowhere to be seen[^3]).
```
Comment (MsSlacker) [inline _posts/2019-12-19-sw-ix-review.md]: Sith, not Siths

**Review by MsSlacker — CHANGES_REQUESTED** [review]

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**